src/styles/HomepageItem.css

```diff
@@ -0,0 +1,54 @@
+.item {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 160px;
+  background-color: rgba(54, 37, 114, 0.518);
+  border-radius: 10px;
+  text-align: center;
```

Comment:
- [ ] Good job making most of the changes @waelelsafty07, but for the home page, as suggested by the previous reviewer, we need to choose two major colors and stick to those colors. currently, there are (white, black, red, and light brown) colors on your design, beside that the background white is still visible, this is a bit odd compared to the original design. As you can see from the original design there is no margin between elements. Please try to remove the white background and stick to the original design. 

### Your design 
![image](https://github.com/waelelsafty07/Metrics-webapp/assets/69077061/e4177146-b3f4-4ddb-bbb5-cd9e54a65c86)
### Original design 

![](https://user-images.githubusercontent.com/91486791/249161818-7de0cd23-14cc-40e6-aaf4-4b3a5797ee5d.png)